geotags/views.py

```diff
@@ -47,16 +49,18 @@
             ORDER BY sounds_sound.created DESC
                      %(end)s"""
 
-def generate_json(sound_queryset):
-    # When using Django Sound queries:
-    #sounds_data = [[s.id, s.geotag.lat, s.geotag.lon] for s in sound_queryset]
+def generate_bytearray(sound_queryset):
+    # sounds as bytearray
+    packed_sounds = cStringIO.StringIO()
+    for sound in sound_queryset:
+        packed_sounds.write(struct.pack("i", sound.id))
+        packed_sounds.write(struct.pack("i", int(sound.lat*1000)))
```

Comment:
I think 1000 is too low since we allow full zoom in, we should use at least 10000 or 1000000

'sound' is the output of the raw query, in the old code was: https://github.com/MTG/freesound/blob/master/geotags/views.py#L54
So I think it's OK